proj-sys/build.rs

```diff
@@ -3,13 +3,67 @@ use cmake;
 use flate2::read::GzDecoder;
 use std::fs::File;
 
+#[cfg(target_env = "msvc")]
+use vcpkg;
+#[cfg(target_env = "msvc")]
```

Comment:
Can you put these msvc specific `use` statements within the `try_vcpkg` method body?